v2.1/export.md

```diff
@@ -0,0 +1,159 @@
+---
+title: EXPORT
+summary: Export tabular data from CockroachDB cluster in CSV format.
+toc: false
+---
+
+The `EXPORT` [statement](sql-statements.html) exports tabular data to CSV files.
+
+{{site.data.alerts.callout_danger}}The <code>EXPORT</code> feature is only available to <a href="https://www.cockroachlabs.com/product/cockroachdb/">enterprise</a> users. Also note that this feature is currently under development and is slated for full release in CockroachDB 2.1. The feature flags and behavior are subject to change. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Export File Location
+
+You can use remote cloud storage (Amazon S3, Google Cloud Platform, etc.) to store the exported CSV data. Alternatively, you can use an [HTTP server](create-a-file-server.html) accessible from all nodes.
+
+For simplicity's sake, it's **strongly recommended** to use cloud/remote storage for the data you want to export. Local files are supported; however, they must be accessible identically from all nodes in the cluster.
+
+## Performance
+
```

Comment:
Eh, the more I think about it, the less convinced I am we need this disclaimer. EXPORT doesn't do any local IO and it is obvious it runs a SELECT, which doesn't have a similar warning on it, so I'm not sure it needs a special warning.